src/cls/IPM/Main.cls

```diff
@@ -1083,7 +1083,12 @@ ClassMethod GetListModules(pNamespace As %String = {$Namespace}, pSearch As %Str
       Set ns = $Order(listNS(ns)) 
       Quit:ns=""
       Kill listModules
-      Do ..GetListModules(ns, pSearch, .listModules, pExtraFields, pRepository)
+      Try {
+        // It is possible that GetListModules will fail in some namespaces. Allow it to do so silently.
```

Comment:
Enumerate at least one reason it could fail and why it is safe to ignore. Should we at least report which namespaces it failed in but we expected IPM to be present in?